test/cqlpy/test_keyspace.py

```diff
@@ -188,13 +188,13 @@ def test_alter_keyspace(cql, this_dc, scylla_only):
 # Test trying to ALTER RF of tablets-enabled KS by more than 1 at a time
 def test_alter_keyspace_rf_by_more_than_1(cql, this_dc):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        with pytest.raises(InvalidRequest):
+        with pytest.raises((InvalidRequest, ConfigurationException)):
```

Comment:
Note that this test (and the following one) is probably no longer relevant. As the comment explains, this test tries to change RF from 1 to 3, and the problem we expected is that changing specifically to 3 was not allowed - but now you'll get a completely different error.